src/App.tsx

```diff
@@ -2,8 +1,25 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App: React.FC = () => {
+export type Movie = (typeof moviesFromServer)[0];
```

Comment:
Verify that the `Movie` type correctly represents the structure of the movie objects from `moviesFromServer`. Ensure that all necessary fields are included in the type definition.